packages/roosterjs-editor-dom/lib/metadata/metadata.ts

```diff
@@ -57,9 +57,22 @@ export function setMetadata<T>(element: HTMLElement, metadata: T, def?: Definiti
 }
 
 /**
- * Remove metadata from the given element if any
+ * Remove specific metadata from the given element if any
  * @param element The element to remove metadata from
+ * @param metadata The metadata tag to remove
```

Comment:
I understand you'd like to allow delete a single item of metadata, so we'd better name it `metadataName` or `metadataKey`.